pkg/log/zap/zap.go

```diff
@@ -251,6 +294,7 @@ func (o *Options) BindFlags(fs *flag.FlagSet) {
 // UseFlagOptions configures the logger to use the Options set by parsing zap option flags from the CLI.
 //  opts := zap.Options{}
 //  opts.BindFlags(flag.CommandLine)
+//  flag.Parse()
 //  log := zap.New(zap.UseFlagOptions(&opts))
 func UseFlagOptions(in *Options) Opts {
 	return func(o *Options) {
```

Comment:
Just noticed this now, but I don't think we should be calling `o.addDefaults()` in this function. That gets called later by `zap.New()` after ALL of the option functions have been called.